src/networking/Packets/MsgAddPlayer.php

```diff
@@ -38,10 +38,19 @@ class MsgAddPlayer extends GamePacket
 
     /**
      * @since 1.0.0
+     * @deprecated since 1.1, use getPlayerId instead
      */
     public function getPlayerIndex(): int
     {
-        return $this->playerIndex;
+        return $this->getPlayerId();
```

Comment:
```suggestion
        trigger_deprecation('allejo/bzflag-networking.php', '1.1.0', 'Using "%s" is deprecated, use "%s" instead.', 'getPlayerIndex', 'getPlayerId');

        return $this->getPlayerId();
```

Make a call to `trigger_deprecation()` here please so that dev environments can pick this up in logs.